src/libs/actions/Report.js

```diff
@@ -432,6 +432,67 @@ function unsubscribeFromReportChannel(reportID) {
     Pusher.unsubscribe(pusherChannelName);
 }
 
+/**
+ * Get the report ID, and then the actions, for a chat report for a specific
+ * set of participants
+ *
+ * @param {String[]} participants
+ */
+function fetchOrCreateChatReport(participants) {
+    let reportID;
+
+    if (participants.length < 2) {
+        throw new Error('fetchOrCreateChatReport() must have at least two participants');
+    }
+
+    API.CreateChatReport({
+        emailList: participants.join(','),
+    })
+
+        .then((data) => {
+            if (data.jsonCode !== 200) {
+                alert(data.message);
```

Comment:
Alerts shouldn't be used at all. If anything, I think it's OK to just log this to the console (using `console.error(data.message)` but I would maybe check what other APIs do with a non-200 response and see what they do.